code/renderers/react/src/entry-docs.ts

```diff
@@ -0,0 +1,3 @@
+export { parameters } from './docs/config';
+export { decorators, argTypesEnhancers } from './docs/config';
+export { applyDecorators } from './applyDecorators';
```

Comment:
@tmeasday I'd either like to chat with you about this thing, or have your eyes/approval on this change.

I find it quite difficult to understand what this applyDecorators function is for (this specific one, not in general).

It **seems** to me that it's for docs, for ensuring some decorating applying some annotation is in the correct order.

I moved to it the `entry-docs` because it has references to docs in there:
https://github.com/storybookjs/storybook/blob/a920749409a5b190b3a1e6a6a85250d539a52ce6/code/renderers/react/src/applyDecorators.ts#L5

I'd like to know if this applyDecorators thing has other uses other than moving the `jsxDecorator` (which should only ever be added when docs in enabled).

Reply:
@ndelangen happy to chat. It's from https://github.com/storybookjs/storybook/pull/21907 and you are correct, the only purpose is to move the `jsxDecorator` to the end of the list.

Do we only add the `jsxDecorator` in `entry-docs`? I think we do, so this seems like the right place for it.